src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -120,6 +120,16 @@ public function load(array $configs, ContainerBuilder $container): void
         $patchFormats = $this->getFormats($config['patch_formats']);
         $errorFormats = $this->getFormats($config['error_formats']);
         $docsFormats = $this->getFormats($config['docs_formats']);
+        $jsonSchemaFormats = $config['jsonschema_formats'];
```

Comment:
During my digging though the codebase I noticed a couple of places where `json_schema` is used over `jsonschema`. Should I change this to add the `_`?

Reply:
I like it like this.